blockchain/chain/Cargo.toml

```diff
@@ -36,7 +36,7 @@ interpreter = { path = "../../vm/interpreter/" }
 json = []
 
 [dev-dependencies]
-multihash = "0.10.0"
+multihash = { version = "0.13", default-features = false, features = ["std", "multihash-impl", "blake2b", "sha2", "multihash-impl", "derive"] }
```

Comment:
I don't think you need `multihash-impl` here as you derive your own table (same for the `Cargo.toml` further down.

Reply:
ah ya I forgot to clean up the features, also don't need sha2, thanks for pinging